### ISSUE TITLE
Squiz.WhiteSpace.OperatorSpacingSniff doesn't detect negative numbers correctly

### ISSUE THREAD

**Issue by Konafets** [squizlabs/PHP_CodeSniffer#429]
The sniff doesn't check the right withespace. The following code doesn't raise an error. I would expect that there is no whitespace allowed after the minus operator.

```php
$y = - 10 + 2;
```

**Comment by gsherwood** [comment]
You can use the Squiz.Formatting.OperatorBracket sniff if you want to enforce this. The error message you get is `No space allowed between minus sign and number`.

The OperatorSpacing sniff is only concerned with arithmetic operators and specifically excludes minus signs that are used for making a number negative.

**Comment by Konafets** [comment]
I will look into that. Thanks!

**Comment by carusogabriel** [comment]
@gsherwood I come with this same issue with the following code:

```php
$x = -1;
```

My ruleset in specific forbids "useless" parentheses, witch the `OperatorBracket` enforces them. Is there another way to fix this issue with negative numbers?
